components/eamxx/src/physics/mam/eamxx_mam_optics_process_interface.cpp

```diff
@@ -1,110 +1,598 @@
+                      grid_name);  // vertical pressure velocity
+  add_field<Required>("T_mid", scalar3d_layout_mid, K,
+                      grid_name);  // Temperature
+  add_field<Required>("p_mid", scalar3d_layout_mid, Pa,
+                      grid_name);  // total pressure
+
+  add_field<Required>("p_int", scalar3d_layout_int, Pa,
+                      grid_name);  // total pressure
+  add_field<Required>("pseudo_density", scalar3d_layout_mid, Pa, grid_name);
+  add_field<Required>("pseudo_density_dry", scalar3d_layout_mid, Pa, grid_name);
+
+  add_field<Required>("qv", scalar3d_layout_mid, q_unit, grid_name,
+                      "tracers");  // specific humidity
+  add_field<Required>("qi", scalar3d_layout_mid, q_unit, grid_name,
+                      "tracers");  // ice wet mixing ratio
+  add_field<Required>("ni", scalar3d_layout_mid, n_unit, grid_name,
+                      "tracers");  // ice number mixing ratio
+
+  // droplet activation can alter cloud liquid and number mixing ratios
+  add_field<Updated>("qc", scalar3d_layout_mid, q_unit, grid_name,
+                     "tracers");  // cloud liquid wet mixing ratio
+  add_field<Updated>("nc", scalar3d_layout_mid, n_unit, grid_name,
+                     "tracers");  // cloud liquid wet number mixing ratio
+
+  add_field<Required>("phis", scalar2d_layout_col, m2 / s2, grid_name);
+  add_field<Required>("cldfrac_tot", scalar3d_layout_mid, nondim,
+                      grid_name);  // cloud fraction
+  add_field<Required>("pbl_height", scalar2d_layout_col, m,
+                      grid_name);  // planetary boundary layer height
+
+  // shortwave aerosol scattering asymmetry parameter [-]
+  add_field<Computed>("aero_g_sw_mam4", scalar3d_swbandp_layout, nondim,
+                      grid_name);
+  // shortwave aerosol single-scattering albedo [-]
+  add_field<Computed>("aero_ssa_sw_mam4", scalar3d_swbandp_layout, nondim,
+                      grid_name);
+  // shortwave aerosol optical depth [-]
+  add_field<Computed>("aero_tau_sw_mam4", scalar3d_swbandp_layout, nondim,
+                      grid_name);
 
+  //
   // shortwave aerosol scattering asymmetry parameter [-]
-  add_field<Computed>("aero_g_sw",   scalar3d_swband_layout, nondim, grid_name);
+  add_field<Computed>("aero_g_sw", scalar3d_swbandp_layout2, nondim, grid_name);
   // shortwave aerosol single-scattering albedo [-]
-  add_field<Computed>("aero_ssa_sw", scalar3d_swband_layout, nondim, grid_name);
+  add_field<Computed>("aero_ssa_sw", scalar3d_swbandp_layout2, nondim,
+                      grid_name);
   // shortwave aerosol optical depth [-]
-  add_field<Computed>("aero_tau_sw", scalar3d_swband_layout, nondim, grid_name);
+  add_field<Computed>("aero_tau_sw", scalar3d_swbandp_layout2, nondim,
+                      grid_name);
+
   // longwave aerosol optical depth [-]
   add_field<Computed>("aero_tau_lw", scalar3d_lwband_layout, nondim, grid_name);
+  // // aerosol extinction optical depth
```

Comment:
Is this comment correct? aero_tau_forward is aerosol extinction optical depth? With a name like forward I'd expect this to be something like the forward scattering fraction or something

Reply:
This comment is incorrect. It is updated now; this should be `aerosol forward scattered fraction`.